package.json

```diff
@@ -200,5 +200,22 @@
     "stream": "stream-browserify",
     "vm": "vm-browserify",
     "fs": "react-native-level-fs"
-  }
+  },
+  "resolutions": {
+    "chalk@5.6.1": "5.6.2",
```

Comment:
this is the pattern for replacing affected versions with a new version.

For each of the affected versions, I'm replacing with the latest unaffected version on npm